README.md

```diff
@@ -101,14 +115,32 @@ The following refers to preconditions that need to be met before running `botium
 * to run Botium tests: `botium-cli run`
 * To open up a browser UI with the test cases: `botium-cli emulator browser --convos=./spec/convo`
 
-# Future Things to do
-* Validate same city names and differents country pairings (E.g. Differentiate between Sydney Canada and Sydney Australia)
-* When getting information about uplinks and appliances, we currently have it such that DialogFlow passes information back to the code here. This feels dirty, and we would ideally like to make it such that the information is stored on the server so that it can be retrieved there rather than having to pass it backwards and forwards. 
-
-# Untested/Deprecated Functionalities:
+## Untested/Deprecated Functionalities:
 The following actions have not been tested out and are deprecated. It has been left in the respository for future possible use:
 - Add site to wan
 - Add sites to wan
 - Clear sites
 - Create SSID
 - Create Zone
+
+# To Do
+* Validate same city names and differents country pairings (E.g. Differentiate between Sydney Canada and Sydney Australia)
+* When getting information about uplinks and appliances, we currently have it such that DialogFlow passes information back to the code here. This feels dirty, and we would ideally like to make it such that the information is stored on the server so that it can be retrieved there rather than having to pass it backwards and forwards. 
```

Comment:
It took me a while to remember what this was referring to, it wasn't immediately clear to me. This is the thing where information is stored in this app between two actions, for choosing a specific uplink or appliance, right? Maybe explicitly mention that this app stores the list of appliances/uplinks so the user can choose one in a followup action.

Also, with "we would ideally like to make it such that the information is stored on the server", which server are you referring to?